cis/com.b2international.snowowl.snomed.cis.rest/src/com/b2international/snowowl/snomed/cis/rest/CisBulkSctIdService.java

```diff
@@ -230,6 +230,7 @@
 				.setDescription(jobDescription)
 				.setRequest(request)
 				.setUser(User.SYSTEM.getUsername())
+				.setAutoClean(true)
```

Comment:
Note that jobs that run into a problem will not be removed: https://github.com/b2ihealthcare/snow-owl/blob/f602f1addc56dd466941121c77c3c88768022a0b/core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/jobs/RemoteJob.java#L102-L123

On the `CisBulkJobService` CIS API level, we are offering to return jobs with a `finished` status as well, these will not be available if they are auto-cleaned. The service relies directly on the remote job storage. In case of a bulk generate job, this means the resulting SCTIDs are not available (maybe for registers this can be done because we already know what is being registered).

Reply:
Although someone who monitors progress by polling the CIS bulk jobs API will not see when their registration job finishes. 😕

Reply:
Yeah agree, this is unfortunately against the current CIS API design, so as @nagyo already did, I suggest reverting the changes and using a better alternative like explicit clean-up tasks or GC style just in time cleanups (when a new request comes in the system will try to detect and clean any obsolete remote jobs automatically).